wd_capabilities.au3

```diff
@@ -145,93 +153,121 @@ EndFunc   ;==>_WD_CapabilitiesStartup
+		$v_WatchPoint = @ScriptLineNumber
 		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__SPECIFICVENDOR__OPTS)
 		$s_Notation &= '[' & $key & ']'
-	ElseIf $key = 'args' Then ; for adding "args" capability in specific/vendor capabilities : ........
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__SPECIFICVENDOR__ARGS)
-		__WD_CapabilitiesSwitch($key, $value1, $value2)          ; as the notation was modified now parameters should be switched
-		If $value1 Then
-			$value1 = $key & '=' & $value1
-		Else
-			$value1 = $key
+		Local $iCurrent2 = UBound(Json_Get($_WD_CAPS__OBJECT, $s_Notation))
+		SetError(0)
+		$s_Notation &= '[' & $iCurrent2 & ']' ; here is specified which one of JSON ARRAY element should be used
+		If $value2 Then
+			$v_WatchPoint = @ScriptLineNumber
+			$value1 &= '=' & $value2
 		EndIf
-		If Not @Compiled Then __WD_ConsoleWrite("- IFNC: " & @ScriptLineNumber & ' $s_Notation =' & $s_Notation)
-		If Not @Compiled Then __WD_ConsoleWrite("- IFNC: " & @ScriptLineNumber & ' $s_Notation =' & $s_Notation & ' = ' & $value1)
-	ElseIf $key = 'prefs' Then ; for adding "prefs" capability in specific/vendor capabilities : ........
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__SPECIFICVENDOR__PREFS)
-		__WD_CapabilitiesSwitch($key, $value1, $value2)          ; as the notation was modified now parameters should be switched
-		$s_Notation &= '[' & $key & ']'
-;~ 		If Not @Compiled Then __WD_ConsoleWrite("- IFNC: " & @ScriptLineNumber & ' $s_Notation =' & $s_Notation)
-	ElseIf $key = 'log' Then ; for adding "log" capability in specific/vendor capabilities : ........
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__SPECIFICVENDOR__LOG)
-		__WD_CapabilitiesSwitch($key, $value1, $value2)          ; as the notation was modified now parameters should be switched
-		$s_Notation &= '[' & $key & ']'
-;~ 		If Not @Compiled Then __WD_ConsoleWrite("- IFNC: " & @ScriptLineNumber & ' $s_Notation =' & $s_Notation)
-	ElseIf $key = 'env' Then ; for adding "env" capability in specific/vendor capabilities : ........
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__SPECIFICVENDOR__ENV)
-		__WD_CapabilitiesSwitch($key, $value1, $value2)          ; as the notation was modified now parameters should be switched
-		$s_Notation &= '[' & $key & ']'
-;~ 		If Not @Compiled Then __WD_ConsoleWrite("- IFNC: " & @ScriptLineNumber & ' $s_Notation =' & $s_Notation)
-	ElseIf $value2 = '' Then ; for string/boolean value type in standard capability : https://www.w3.org/TR/webdriver/#capabilities
-		$s_Notation = __WD_CapabilitiesNotation($_WD_CAPS__STANDARD__STRINGORBOOL)
-		$s_Notation &= '[' & $key & ']'
+
+	ElseIf StringRegExp($key, $_WD_CAPS_TYPES__SPECIFICVENDOR_OBJECT, $STR_REGEXPMATCH) Then ; for string/boolean value type in standard capability : https://www.w3.org/TR/webdriver/#capabilities
```

Comment:
"string/boolean" again

Reply:
should be fixed ...... in soon in next commit

Reply:
> 	ElseIf StringRegExp($key, $_WD_CAPS_TYPES__SPECIFICVENDOR_OBJECT, $STR_REGEXPMATCH) Then ; for string/boolean value type in standard capability : https://www.w3.org/TR/webdriver/#capabilities

This comment wasn't modified and still doesn't match IMO